### PR TITLE
Add possibility to swap images

### PR DESCRIPTION
**Pull request by tirli**
Add a possibility to swap images. The swap button appears only when both layers available.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,11 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 
 - ADDED: Allows Leaflet version 0.7.7 through 1.x
 
+## [v2.1.0] - 2020-10-01
```
Comment (lockettks) [inline CHANGELOG.md:10]: How is this file used?

Comment (tirli) [reply CHANGELOG.md:10]: It is just documentation. Decided to add this here for consistency. And I thought that GitHub supports this kind of file to show in the interface, but seems like I'm wrong

**Comment by philvarner** [comment]
FYI, I just setup a circleci config for this project.

**Review by lockettks — APPROVED** [review]
lgtm

**Pull request merged**